images/server/Dockerfile.fedora

```diff
@@ -23,7 +23,10 @@ RUN dnf install -y \
     samba-winbind \
     samba-winbind-clients \
     tdb-tools \
+    ctdb \
     && dnf clean all \
+    && cp --preserve=all /etc/ctdb/functions /usr/share/ctdb/functions \
+    && cp --preserve=all /etc/ctdb/notify.sh /usr/share/ctdb/notify.sh \
```

Comment:
what's the purpose of these copies? Is the rpm wrong?

Reply:
I wouldn't say wrong per-se, but since we (can) mount an emptydir over /etc/ctdb we'd lose access to those files. This copy makes sure we preserve them in a place that we're unlikely to mount over in a container environment. That said I do think it would be better if ctdb rpm put those not-strictly-configuration files in /usr. :-)

Reply:
makes sense.